specification/billingbenefits/resource-manager/Microsoft.BillingBenefits/stable/2022-11-01/billingbenefits.json

```diff
@@ -0,0 +1,1914 @@
+      "properties": {
+        "displayName": {
+          "$ref": "#/definitions/DisplayName"
+        },
+        "savingsPlanOrderId": {
+          "type": "string",
+          "description": "Identifier of the savings plan created",
+          "readOnly": true
+        },
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState"
+        },
+        "billingScopeId": {
+          "$ref": "#/definitions/BillingScopeId"
+        },
+        "term": {
+          "$ref": "#/definitions/BenefitTerm"
+        },
+        "billingPlan": {
+          "$ref": "#/definitions/BillingPlan"
+        },
+        "appliedScopeType": {
+          "$ref": "#/definitions/AppliedScopeType"
+        },
+        "appliedScopeProperties": {
+          "$ref": "#/definitions/AppliedScopeProperties"
+        },
+        "commitment": {
+          "$ref": "#/definitions/Commitment"
+        }
+      }
+    },
+    "BillingScopeId": {
+      "type": "string",
+      "description": "Subscription that will be charged for purchasing SavingsPlan"
+    },
+    "BillingProfileId": {
+      "type": "string",
+      "readOnly": true,
+      "description": "Fully-qualified identifier of the billing profile where the savings plan is applied. Present only for Field-led or Customer-led customers."
+    },
+    "CustomerId": {
+      "type": "string",
+      "readOnly": true,
+      "description": "Fully-qualified identifier of the customer where the savings plan is applied. Present only for Partner-led customers."
+    },
+    "BillingAccountId": {
+      "type": "string",
+      "readOnly": true,
+      "description": "Fully-qualified identifier of the billing account where the savings plan is applied. Present only for Enterprise Agreement customers."
+    },
+    "ReservedResourceType": {
+      "type": "string",
+      "description": "The type of the resource that is being reserved.",
+      "enum": [
+        "ComputeSavingsPlan",
+        "VirtualMachines",
+        "SqlDatabases",
+        "SuseLinux",
+        "CosmosDb",
+        "RedHat",
+        "SqlDataWarehouse",
+        "VMwareCloudSimple",
+        "RedHatOsa",
+        "Databricks",
+        "AppService",
+        "ManagedDisk",
+        "BlockBlob",
+        "RedisCache",
+        "AzureDataExplorer",
+        "MySql",
+        "MariaDb",
+        "PostgreSql",
+        "DedicatedHost",
+        "SapHana",
+        "SqlAzureHybridBenefit",
+        "AVS",
+        "DataFactory",
+        "NetAppStorage",
+        "AzureFiles",
+        "SqlEdge",
+        "VirtualMachineSoftware"
+      ],
+      "x-ms-enum": {
+        "name": "ReservedResourceType",
+        "modelAsString": true
+      }
+    },
+    "ProvisioningState": {
+      "type": "string",
+      "description": "Provisioning state",
+      "readOnly": true,
+      "enum": [
+        "Creating",
+        "PendingBilling",
+        "ConfirmedBilling",
+        "Created",
+        "Succeeded",
+        "Cancelled",
```

Comment:
>Cancelled [](http://example.com/codeflow?start=9&length=9)

Canceled

Reply:
Synced offline, we are adding suppression for this